contrib/localnet/scripts/genesis-stateful.sh

```diff
@@ -91,7 +91,7 @@ then
 
 # 2. Add the observers , authorizations and required params to the genesis.json
   zetacored collect-observer-info
-  zetacored add-observer-list
+  zetacored add-observer-list --keygen-block 55
```

Comment:
Can we use a lower height to speed up the process ?

Reply:
I was failing with the previous number, 20.
I reuse the same as one the regular test.

Reply:
20 seems actually to be fine. Maybe recents change in the tests fix this. Let me do some more testing and maybe change this value everywhere.